File: CPP/Program/Komponen.cpp
#pragma once
#include <bits/stdc++.h>
using namespace std;

class Komponen {
    // Attributes
    private:
        string merk;
        string nama;
    
    // Constructor
    public:
        // Default constructor
        Komponen() {
            this->merk = "";
            this->nama = "";
        }
        
        // Parameterized constructor
        Komponen(string merk, string nama) {
            this->merk = merk;
            this->nama = nama;
        }
        
        // Getters
        string getMerk() {
            return this->merk;
        }
        
        string getNama() {
            return this->nama;
        }
        
        // Setters
        void setMerk(string merk) {
            this->merk = merk;
        }
        
        void setNama(string nama) {
            this->nama = nama;
        }
        
        // Destructor
        ~Komponen() {
        }
};

File: Java/Program/Komponen.java
public class Komponen {
    // Attributes
    private String merk;
    private String nama;
    
    // Constructors
    public Komponen() {
        this.merk = "";
        this.nama = "";
    }
    
    public Komponen(String merk, String nama) {
        this.merk = merk;
        this.nama = nama;
    }
    
    // Getters
    public String getMerk() {
        return this.merk;
    }
    
    public String getNama() {
        return this.nama;
    }
    
    // Setters
    public void setMerk(String merk) {
        this.merk = merk;
    }
    
    public void setNama(String nama) {
        this.nama = nama;
    }
}
